[BUG] ubuntu 배포 서버에서 yarn start 불가

## 👉  버그 설명
- CI/CD 배포 시 
> error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.

에러 발생.


<br>

## ✅  해결 방법
- 버그를 어떻게 해결했는지 적어주세요.
1. 원인 
  - npm install yarn으로 yarn을 설치할 때 yarn이 아닌 cmdtest가 설치되었을 것으로 보임(현재 ubuntu 서버 키가 없는 노트북으로 진행 중이라 정확한 원인이 확인 될 시 수정하겠습니다)

2. 방법
>sudo apt remove cmdtest
sudo apt remove yarn
curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
sudo apt-get update
sudo apt-get install yarn -y

  a. 설치되었을 것으로 보이는 cmdtest 제거
  b. yarn 제거
  c. [yarn 공식 문서](https://classic.yarnpkg.com/en/docs/install#debian-stable)에서 권장하는 ubuntu yarn 설치 방법대로 설치
  d. 실행

4. 결과
  - CI/CD 배포가 잘 동작합니다

<br>

## 📷  스크린 샷
- 가능한 경우 문제를 파악하는 데 도움이 되는 스크린 샷을 추가해주세요.

<br>

## ⭐ 추가 내용
- [[Yarn install command error No such file or directory: 'install'](https://stackoverflow.com/questions/46013544/yarn-install-command-error-no-such-file-or-directory-install)](https://stackoverflow.com/questions/46013544/yarn-install-command-error-no-such-file-or-directory-install)
- [Installation](https://classic.yarnpkg.com/en/docs/install#debian-stable)

--- a/deploy.sh
+++ b/deploy.sh
@@ -26,6 +26,16 @@ export NVM_DIR="$HOME/.nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
 [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
 
+# 임시 작업
+# 현재 ubuntu 서버에 접속할 수 있는 키가 없어 임시적으로 작업
+
+sudo apt remove cmdtest
+sudo apt remove yarn
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+sudo apt-get update
+sudo apt-get install yarn -y
+
 yarn install
 
-nuhup yarn start >> $REPOSITORY/nohup.out 2>&1 &
+nohup yarn start >> $REPOSITORY/nohup.out 2>&1 &

